src/main/java/ch/njol/skript/Skript.java

```diff
@@ -923,6 +924,18 @@ public void run() {
 			}
 		}, this);
 
+		// Send a warning to console when the plugin is reloaded
+		Bukkit.getPluginManager().registerEvents(new Listener() {
+			@EventHandler
+			public void onServerReload(ServerLoadEvent event) {
+				if (!(event.getType() == ServerLoadEvent.LoadType.RELOAD)) return;
+
+				Skript.warning("Looks like you've used /reload or used a plugin to reload Skript. This is not supported behaviour.");
+				Skript.warning("Reloading may cause issues.");
+				Skript.warning("Please consider restarting your server.");
```

Comment:
```suggestion
				Skript.warning("It appears that /reload or another plugin reloaded Skript. This is not supported behaviour and may cause issues.");
				Skript.warning("Please consider restarting the server instead.");
```
Less personal and targeted. Also, who should this be sent to? Just console or do you think it should send to opped players as well?